File: Leetcode/27-RemoveElement/RemoveElement.py
import typing
from typing import List
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        actualsize = 0
        if not nums:
            return 0
        for i in range(len(nums)) :
            if nums[i]!= val :
                nums[actualsize] = nums[i]
                actualsize = actualsize + 1
        return actualsize
nums = [3,2,2,3]
s = Solution()
print(s.removeElement(nums,3))

File: Leetcode/27-RemoveElement/RemoveElement.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int actualsize=0;
        for(int i =0; i<nums.length; i++){
            if(nums[i]!=val){
                nums[actualsize]=nums[i];
                actualsize++;
            }
        }
        return actualsize;
    }
}
public class RemoveElement{
    public static void main(String[] args) {
        Solution s = new Solution();
        int [] nums = {3,2,2,3};
        int val = 3;
        System.out.println((s.removeElement(nums, val)));
    }
}